cmd/kubelet/app/options/container_runtime.go

```diff
@@ -39,20 +35,8 @@ var (
 // NewContainerRuntimeOptions will create a new ContainerRuntimeOptions with
 // default values.
 func NewContainerRuntimeOptions() *config.ContainerRuntimeOptions {
-	dockerEndpoint := ""
-	if runtime.GOOS != "windows" {
-		dockerEndpoint = "unix:///var/run/docker.sock"
-	}
-
 	return &config.ContainerRuntimeOptions{
-		ContainerRuntime:          kubetypes.DockerContainerRuntime,
-		DockerEndpoint:            dockerEndpoint,
-		DockershimRootDirectory:   "/var/lib/dockershim",
-		PodSandboxImage:           defaultPodSandboxImage,
-		ImagePullProgressDeadline: metav1.Duration{Duration: 1 * time.Minute},
-
-		CNIBinDir:   "/opt/cni/bin",
-		CNIConfDir:  "/etc/cni/net.d",
-		CNICacheDir: "/var/lib/cni/cache",
+		ContainerRuntime: kubetypes.DockerContainerRuntime,
```

Comment:
need to set it to  `kubetypes.RemoteContainerRuntime`

Reply:
@adisky thanks, I miss it